General language improvements (#333)
Meeds-io/whitepaper: en/2-deeds.md

## 2.1. Deed NFTs

> In the digital realm, Non-Fungible Tokens (NFTs) serve a similar purpose to deeds in the physical world. Like a deed confirming tangible property ownership, NFTs establish indisputable ownership of digital assets. This ownership is securely recorded on a blockchain, ensuring each asset is unique and non-replicable.

Reviewer: In the digital realm, Non-Fungible Tokens (NFTs) function akin to physical world deeds, confirming ownership of digital assets securely recorded on a blockchain. They establish indisputable ownership, akin to tangible property deeds, ensuring each asset's uniqueness and non-replicability.

In the digital realm, Non-Fungible Tokens (NFTs) serve a similar purpose to deeds in the physical world. Like a deed confirming tangible property ownership, NFTs establish indisputable ownership of digital assets. This ownership is securely recorded on a blockchain, ensuring each asset is unique and non-replicable.

> Within the WoM, a Deed is conceptualized as an NFT, representing the foundational unit of ownership. Holding a Deed empowers its owner to manage a Community Hub and makes them eligible for receiving Mintium rewards, a form of recognition and incentive within the WoM ecosystem (further details in section 2.6).

Reviewer 2: In the context of WoM, a Deed is conceptualized as an NFT, representing the foundational unit of ownership. Owning a Deed empowers individuals to manage a Community Hub and qualifies them for Mintium rewards—a form of recognition and incentive within the WoM ecosystem (refer to section 2.6 for additional details).

> Within the WoM, a Deed is conceptualized as an NFT, representing the foundational unit of ownership. Holding a Deed empowers its owner to manage a Community Hub. It makes them eligible to receive Mintium rewards, a form of recognition and incentive within the WoM ecosystem (further details in section 2.6).

Author: ```suggestion
Within the WoM, a Deed is conceptualized as an NFT, representing the foundational unit of ownership. Holding one empowers its owner to manage a __Community Hub__. Hubs are eligible to receive Mintium rewards, a form of sponsorship for communities that manage to genrate authentic engagement from their contributors (further details in section 2.6).
```

> The WoM is a diverse and inclusive platform. It welcomes various organizations, from DAOs, traditional businesses, and associations to think tanks, theater companies, project groups, and educational institutions. In this digital landscape, a Deed symbolizes the organization's digital presence and footprint in WoM, functioning as a bridge between real-world operations and digital endeavors.

Author: ```suggestion
The WoM is a diverse and inclusive platform. It welcomes various organizations, from DAOs, traditional businesses, and associations to think tanks, theater companies, project groups, and educational institutions. In this digital landscape, a Deed symbolizes the organization's digital presence and footprint. I works as a bridge between real-world operations and digital endeavors.
```

## 2.2. Deeds supply

> The WoM will ultimately supply a limited amount of Deeds. Each Deed includes a predefined maximum number of users and offers a reward capacity (Minting Power). The term' user' here and hereafter designates any employee, worker, freelancer, volunteer, etc.

Author: ```suggestion
The WoM will ultimately supply a limited amount of Deeds. Each Deed includes a predefined maximum number of users and offers a reward capacity (Minting Power). The term __'user'__ here and hereafter designates any employee, worker, freelancer, volunteer, etc.
```

## 2.3. Deeds ownership

A Deed gives its owner the right to USE, RENT, or SELL it.

- USE – The owner can use the Deed to run a hub for his organization.
- RENT - The owner can also rent the Deed to another organization and earn passive income.
- SELL - The owner can sell it on a secondary NFT marketplace such as [OpenSea](https://opensea.io).

Being an active holder or renter of a Deed, the organization is entitled to MEED rewards according to the "Mintium" model described below (see 2.6).

## 2.4. Community Hubs

Organizations seek to attract and retain top talent in today's competitive market through compensation and enhancing work engagement. The WoM addresses this challenge with a unique, tokenized reward system embodied in Community Hubs.

> Within these Hubs, community leaders can drive members' engagement by offering a transparent menu of incentives for work items. Participants earn engagement points for their efforts. The system then periodically converts these points into reward tokens per a predetermined budget.

Author: ```suggestion
Within these Hubs, community leaders can drive members' engagement by offering a transparent menu of incentivized work items. Participants earn engagement points for their efforts. Then the system periodically converts these points into reward tokens per a predetermined budget.
```

> Community Hubs offer a suite of tools and features designed to boost and manage contributor engagement:

Reviewer 3: Simplified English, but the word"offer" should be replaced with "offers"

Author: > Simplified English, but the word"offer" should be replaced with "offers"

No. "Community Hubs offer"  plural

- **Actions**: Incentivized tasks guide contributors, aligning their efforts with organizational objectives and boosting motivation and direction.
- **Gamification**: Leaderboards and badges transform work into a fun, competitive experience, enhancing engagement and enjoyment.

> - **Engagement Programs**: Customized programs with specific targets and rewards clarify organizational goals and somewhat recognize contributor achievements.

Author: ```suggestion
- **Engagement Programs**: Customized programs with specific targets and rewards clarify organizational goals and fairly recognize contributor achievements.
```

- **Kudos**: Immediate peer recognition boosts morale and fosters a culture of appreciation, enhancing team cohesion and motivation.
- **Spaces**: Organized sub-groups and projects streamline communication and collaboration, increasing collective productivity.
- **Wallet**: The Wallet app securely stores MEED rewards, representing user contributions, and enables easy transactions within the Hub and on the broader network.
- **Perks**: Various rewards, from gift cards to NFTs, incentivize ongoing engagement and foster loyalty within the organization's ecosystem.
- **Tasks**: Streamlined project and task management tools boost productivity by simplifying workflows and tracking progress.
- **Notes**: A handy notebook app for documenting essential information like onboarding guides promotes knowledge sharing and organizational self-sufficiency.
- **Branding**: Customizable Hub appearance attracts new contributors and strengthens their connection to the organization's identity.
- **Analytics**: Insightful analytics into community engagement enable leaders to identify effective strategies and make informed decisions for optimization.

**Starting a Hub**

> Initiating a Hub is most seamless via meeds.io, which offers access to a cloud platform managed by the Meeds Association. However, the WoM caters to decentralization. Since the Meeds software is open source and freely distributed, organizations can self-host a Hub server and connect to the WoM network, granting them more control and customization in their Hub's setup and management.

Author: ```suggestion
Initiating a Hub is most seamless via meeds.io, which offers access to a  managed cloud service. However, the WoM caters to decentralization. Since the Meeds software is open source and freely distributed, organizations can self-host a Hub server and connect to the WoM network, granting them more control and customization in their setup and management.
```

**Capturing Work Signals**
Hubs can integrate with various platforms like GitHub, Twitter, and Discord and are extendable via API to connect with services such as CRM software, accounting software, and clocking terminals. This extensibility allows for diverse professional contributions to be recognized and rewarded. All connectors undergo a verification process to guarantee integrity and functionality.

**Enriching the Perks Store**
Organizations can tailor the store with custom perks, including training courses and gym memberships, creating a self-reinforcing cycle of rewards and engagement. This internal store further incentivizes contributors and fosters a deeper connection with the organization's ecosystem.

**Engagement and Rewards**

> Active participation in the Hub contributes to an organization's global engagement score. This metric reflects the effectiveness of engagement strategies, considering factors like user activity levels and the equitable distribution of work and rewards. This publicly auditable score entitles Hubs to a distribution of MEED tokens minted from the protocol, serving as a measure of the effectiveness of their engagement strategies.

Author: ```suggestion
Active participation contributes to an organization's global engagement score. This metric reflects the effectiveness of engagement strategies, considering factors like user activity levels and the equitable distribution of work and rewards. This publicly auditable score entitles Hubs to a distribution of MEED tokens minted from the protocol, serving as a measure of the effectiveness of their engagement strategies.
```

> Community Hubs in the WoM ecosystem aim to become central to community life, integrating internal communications, real-time interactions, governance, and various productivity tools. These enhancements will ensure the Hubs remain interconnected with the best tools organizations employ. They are a powerful testament to Meeds' commitment to fostering a vibrant, engaged community.

Author: ```suggestion
Hubs aim to become central to community life, integrating internal communications, real-time interactions, governance, and various productivity tools. These enhancements will ensure they remain interconnected with the best tools organizations employ. They are a powerful testament to Meeds' commitment to fostering a vibrant, engaged community.
```

## 2.5. Rewarding Model

> The WoM incentivizes and gamifies user engagement across various work activities tailored to each community Hub's unique programs. Users earn points reflecting their active participation, with each community determining its own rules for point allocation.

Author: ```suggestion
The WoM incentivizes and gamifies user engagement across various work activities tailored to each unique programs. Users earn points reflecting their active participation, with each community determining its own rules for point allocation.
```

For example, an IT company might award 10 points for coding on GitHub, whereas a digital agency could place a higher value on social media engagement, like retweets, with similar points.

Regular reward distributions occur as contributors in each Hub earn the points. Users automatically receive MEED (or other) tokens in their wallets. 

> While MEED tokens are the standard form of reward, Hubs have the flexibility to offer different types of compensation. In that case, the protocol applies a nominal fee of 5%. This fee contributes to the Meeds treasury, playing a crucial role in maintaining the sustainability of the ecosystem.

Author: ```suggestion
While MEED tokens are the standard form of reward, Hubs have the flexibility to offer different types of compensation. In that case, the protocol applies a nominal fee of 5%. This fee contributes to the treasury of the DAO, playing a crucial role in maintaining the sustainability of the ecosystem.
```

## 2.6. The "Mintium" model – tokenizing work engagement

By engaging their users in the WoM, communities get compensated through an innovative mintium model.

> Engagement points gained by the Hubs' users participate in the global engagement index, calculated by the Meeds Association. The index uses the same principle for all Hubs. The differences in point values allocated to actions by communities do not influence the global index computation.

Author: ```suggestion
Engagement points gained by the users participate in the global engagement index. The differences in point values allocated to actions by communities do not influence the global index computation.
```

> The WoM allocates some MEED tokens each week to the global engagement minting. It's computed weekly, pro-rata, among the participating Hubs. Some Hubs get an additional boost depending on their Deed type. The Hubs that claim their MEED tokens over the week receive them. The unclaimed MEED tokens augment the allocation for next week.

Author: ```suggestion
The WoM allocates some MEED tokens each week to the global engagement pool. It's computed weekly, pro-rata, among the participating Hubs. Some Hubs get an additional boost depending on their Deed type. The Hubs that claim their MEED tokens over the week receive them. The unclaimed MEED tokens augment the allocation for next week.
```

## 2.3. Deeds ownership

> A Deed gives its owner the right to USE, RENT, or SELL it.

Reviewer 4: ```suggestion
A Deed gives its owner the right to USE, RENT or SELL it.
```

- USE – The owner can use the Deed to run a hub for his organization.
- RENT - The owner can also rent the Deed to another organization and earn passive income.
- SELL - The owner can sell it on a secondary NFT marketplace such as [OpenSea](https://opensea.io).

Being an active holder or renter of a Deed, the organization is entitled to MEED rewards according to the "Mintium" model described below (see 2.6).

## 2.4. Community Hubs

> Organizations seek to attract and retain top talent in today's competitive market through compensation and enhancing work engagement. The WoM addresses this challenge with a unique, tokenized reward system embodied in Community Hubs.

Reviewer 5: ```suggestion
Organizations seek to attract and retain top talent in today's competitive market through compensation and enhancing work engagement. The WoM addresses this challenge with a distinctive, tokenized reward system incorporated in Community Hubs.
```

Reviewer 5: Using proper-sounding words that look professional and appealing gives this sentence a catchy look/feeling. Just like the words that were used in this sentence to replace the previous ones.
